posts/2021-05-06-Rust-1.52.0.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: "Announcing Rust 1.52.0"
+author: The Rust Release Team
+release: true
+---
+
+The Rust team is happy to announce a new version of Rust, 1.52.0. Rust is a
+programming language that is empowering everyone to build reliable and
+efficient software.
+
+If you have a previous version of Rust installed via rustup, getting Rust
+1.52.0 is as easy as:
+
+```console
+rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install]
+from the appropriate page on our website, and check out the
+[detailed release notes for 1.52.0][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1510-2021-03-25
+
+## What's in 1.52.0 stable
+
+This release is fairly small, primarily containing stabilizations of various APIs in the standard library.
+
+This release also closes out a long-standing bug in the Rust compiler: forward progress is no longer assumed by LLVM, which means that some programs which used to compile incorrectly are now properly handled. This is due to changes in LLVM IR semantics, which no longer assume forward progress is guaranteed and optimize for that.
```

Comment:
"forward progress" is a little jargony - I would add an example of code that used to be miscompiled and mention that it's related to infinite loops. Maybe link to https://github.com/rust-lang/rust/issues/28728 as well?